packages/vue/src/components/organisms/SfFooter/SfFooter.js

```diff
@@ -43,6 +43,10 @@ export default {
     }
   },
   methods: {
+    updateItems(title) {
```

Comment:
what is the purpose of this method? does it come from some other PR?

Reply:
it's a part of small SfFooter refactor, I remove it for this PR https://github.com/DivanteLtd/storefront-ui/pull/472/commits/e0f863e7748b2252697ce0801af83e3bbb83da73